x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/components/package_card.tsx

```diff
@@ -247,7 +260,18 @@ export function PackageCard({
         }
         onClick={onClickProp ?? onCardClick}
       >
-        <EuiFlexGroup gutterSize="xs" wrap={true}>
+        <EuiFlexGroup
+          gutterSize="xs"
+          wrap={true}
+          css={css`
+            width: ${installationStatusVisible
+              ? `calc(100% - ${theme.euiTheme.base * 4}px)`
+              : '100%'};
+            overflow: hidden scroll;
```

Comment:
I don't think scroll on overflow is a good solution here. It's not at all obvious that there are additional hidden badges, and on some browsers the scrollbar itself will always be visible on these tiny elements which is visual clutter.

can we just show all badges and let the card expand to whatever height it needs to be?

Reply:
@jen-huang, Thanks for the feedback. I've updated the styles, wdyt? https://github.com/elastic/kibana/pull/236184/commits/c749c2352cecd8a5397ded516b34f0121faf3458

Extreme cases:

https://github.com/user-attachments/assets/50d82a81-4181-4a87-b5a0-1c6cf225027f

Common cases:

https://github.com/user-attachments/assets/baffe1ea-9e4e-4c0c-ba0b-c0f749afb11b

Reply:
@angorayc thanks for the changes! in the test deployment I see the Integrations browse grid overflowing like this on a narrow viewport:

<img width="773" height="511" alt="image" src="https://github.com/user-attachments/assets/a34dd5c0-e8a9-45e2-b504-19c510cdbb3d" />

it's possible that this existed prior to this PR though. could you test the extreme cases on this page? if it also happens without this PR then don't worry about it :)

Reply:
Thanks for catching this @jen-huang , I've updated it again. 


https://github.com/user-attachments/assets/105827ab-3837-462e-912c-0c7d91c50a8a


https://github.com/user-attachments/assets/2d70fb63-5ba7-4729-837c-c072574b3ae3



https://github.com/user-attachments/assets/d5520fac-e36e-4676-ba9c-6f41fa0db7da